model/version.go

```diff
@@ -155,6 +156,18 @@ func (self *Version) SetNotActivated() error {
 	)
 }
 
+func (self *Version) SetAbortedStatus(aborted bool) error {
```

Comment:
can we rename this to just SetAborted as well?

Reply:
done